rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/values.kt

```diff
@@ -725,6 +753,34 @@ class AllValue(val charsToRepeat: String) : Value {
     }
 }
 
+object IsoValue : Value {
+    override fun copy() = this
+
+    override fun toString(): String {
+        return "IsoValue"
+    }
+
+    override fun assignableTo(expectedType: Type): Boolean = true
+
+    override fun asString(): StringValue {
+        TODO("Not yet implemented")
+    }
+}
+
+object JulValue : Value {
+    override fun copy() = this
+
+    override fun toString(): String {
+        return "JulValue"
+    }
+
+    override fun assignableTo(expectedType: Type): Boolean = true
```

Comment:
You are right. I'll change in: 
```kt
override fun assignableTo(expectedType: Type): Boolean = false
```
because `*JUL` is only a flag for conversion.